### PR TITLE
Fulcio: Add support for kmsca backend

### PR DESCRIPTION
**Pull request by nsmith5**
#### Summary

Adds support for the recently added `kmsca` backend in Fulcio

#### Release Note

```release-note
* Added support for `kmsca` backend in Fulcio
```

### CODE REVIEW

File: charts/fulcio/values.schema.json [hunk 1]
```diff
@@ -513,6 +513,26 @@
...
+                            "default": "",
+                            "examples": [
+                                "gcpkms://foo/bar"
+                            ]
+                        },
+                        "kms_cert_chain": {
+                            "$id": "#/properties/server/properties/args/properties/kms_cert_chain",
+                            "type": "string",
+                            "title": "The kms chain schema",
+                            "description": "Certificate chain for KMS CA backend",
```
Comment (haydentherapper) [inline charts/fulcio/values.schema.json:530]: Can we note that the chain starts with the issuing CA and ends with the root CA?

File: charts/fulcio/values.schema.json [hunk 2]
```diff
@@ -513,6 +513,26 @@
...
                             ]
                         },
+                        "kms_resource": {
+                            "$id": "#/properties/server/properties/args/properties/kms_resourse",
+                            "type": "string",
+                            "title": "The kms resource schema",
+                            "description": "The KMS URI for backend KMS",
+                            "default": "",
+                            "examples": [
+                                "gcpkms://foo/bar"
```
Comment (haydentherapper) [inline charts/fulcio/values.schema.json:523]: Can we give the examples for the other backends too, azurekms, hashivault, and awskms?

**Review by haydentherapper — APPROVED** [review]

**Comment by haydentherapper** [comment]
@cpanato Can you kick off the workflow?

**Review by cpanato — APPROVED** [review]
nice one

**Pull request merged**